pages/settings/settings.js

```diff
@@ -6,6 +6,49 @@ var contentTypes = {
 	//humanReadableName: contentType
 	"scripts": "script",
 	"images": "image",
+};
+var keyMap;
+
+settings.get('keyMap', function (val) {
+	keyMap = val;
+	Object.keys(keyMap).forEach(createKeyMapInput);
+});
+
+function createKeyMapInput(action) {
+	var keyMapList = document.getElementById('key-map-list');
+	var key = keyMap[action];
+	var li = document.createElement('li');
+	var label = document.createElement('label');
+	var input = document.createElement('input');
+	label.innerText =	label.htmlFor = action;
```

Comment:
Would it be possible to have the formatted name of the action ("Open Private Tab" instead of "openPrivateTab") here?